src/Microsoft.AspNetCore.Razor/Chunks/Generators/RemoveTagHelperChunkGenerator.cs

```diff
@@ -35,5 +37,23 @@ public override void GenerateChunk(Span target, ChunkGeneratorContext context)
         {
             context.ChunkTreeBuilder.AddRemoveTagHelperChunk(_lookupText, target);
         }
+
+        /// <inheritdoc />
+        public override bool Equals(object obj)
+        {
+            var other = obj as RemoveTagHelperChunkGenerator;
+            return base.Equals(obj) &&
```

Comment:
It would be functionally equivalent to what it is currently but by calling down to base we ensure that any sub-api equals changes are captured here.